domain/build.gradle

```diff
@@ -43,7 +43,8 @@ dependencies {
       'androidx.appcompat:appcompat:1.0.2',
       'androidx.lifecycle:lifecycle-livedata-ktx:2.2.0-alpha03',
       'com.google.dagger:dagger:2.24',
-      "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+      "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version",
+      'com.google.code.gson:gson:2.8.2'
```

Comment:
`Optional Nit`: Alphabetical order

Reply:
Also, why do you need domain layer changes in this PR?

Reply:
Removed